wagtail/test/testapp/migrations/0046_alter_custom_rendition_to_unique_constraint.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
         migrations.AddConstraint(
             model_name="customrendition",
             constraint=models.UniqueConstraint(
-                fields={"image", "filter_spec", "focal_point_key"},
+                fields=("image", "filter_spec", "focal_point_key"),
```

Comment:
I'm not sure if something changed in Django (I'm using 6.0dev), but running `makemigrations` on my machine would sometimes generate a migration for this constraint, with Django thinking it's been modified due to a difference in the set's ordering.